ui/client/components/graph/node-modal/editors/expression/SwitchIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from "classnames"
+import React, {MouseEventHandler} from "react"
+import {useTranslation} from "react-i18next"
+import * as LoaderUtils from "../../../../../common/LoaderUtils"
+
+export default function SwitchIcon(props: Props) {
```

Comment:
We should not rewrite files to .tsx which are not touched in commit

Reply:
I assumed that it's good to commit some more examples of translation by the way